### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by michaelspellacy** [tmpworldwide/tmpworldwide.github.io#7]
TODO: Flesh this out.

**Comment by michaelspellacy** [comment]
Still fleshing out. I'll have more to say on it soon as it has become a hot topic lately.

**Comment by michaelspellacy** [comment]
Okay, so I have been giving this a TON of thought lately. Rather than have accessibility be it's own section (zzzzzzzzzzzzzzzzzz), it should, given it's significant importance, encompass and have a presence withing the _entirety_ of the document. That means each section and update could potentially contain some wisdom surrounding the good old a11y. Thoughts?

**Comment by scampbell44** [comment]
I like this idea. I think this approach more seamlessly integrates the practice of including accessibility elements throughout all our code. It then is simply a standard part of our best practices as it should be.

**Comment by michaelspellacy** [comment]
Did this back in December! Closing!
